elife/scripts/rds-perms2.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
not superiventive either, but given the Salt file is called `postgresql-appdb.sls` it could be called `postgresql-appdb-perms.sh` for consistency (immediately clear that the two are related and makes clear it's not only related to the command line tool `psql` but to the whole `postgresql*.sls` setup)